pkg/rehearse/jobs.go

```diff
@@ -183,6 +203,13 @@ func ConfigureRehearsalJobs(toBeRehearsed config.Presubmits, ciopConfigs config.
 				continue
 			}
 
+			exists, index, templateKey := hasChangedTemplateVolume(rehearsal.Spec.Containers[0].VolumeMounts, rehearsal.Spec.Volumes, templates)
+			if exists {
+				if templateData, err := config.GetTemplateData(templates[templateKey]); err == nil {
+					rehearsal.Spec.Volumes[index].VolumeSource.ConfigMap.Name = config.GetTempCMName(templates[templateKey].Name, []string{templateKey, templateData})
+				}
```

Comment:
If there **was** an error, we should `continue` to avoid submitting possibly broken rehearsal.  If the job uses some changed template but we fail to get its data, there's something wrong.

Reply:
addressed